cmd/k8s-operator/proxy.go

```diff
@@ -223,6 +232,7 @@ func (ap *apiserverProxy) serveExec(w http.ResponseWriter, r *http.Request) {
 		ap.rp.ServeHTTP(w, r.WithContext(whoIsKey.WithValue(r.Context(), who)))
 		return
 	}
+	counterSessionRecordingsAttempted.Add(1) // at this point we know that users intended for this session to be recorded
```

Comment:
nit: I think it would be good for this metric to capture all sessions that were _intended_ to be captured, even if they've misconfigured their network to end up with no recorders available, so maybe we should bump this up a few lines? Although I'm not sure if `determineRecorderConfig` might need to return slightly different information to help us make the decision to increment this metric.

Reply:
As discussed, this is the earliest point where from client's perspective recording has been configured